src/components/spaces/organisms/organelle-wrapper.sass

```diff
@@ -11,13 +13,16 @@
 .hover-location
     position: absolute
     top: 45px
-    right: 5px
+    right: 0
     min-width: 94px
-    height: 18px
-    background: #FFFFFFAA
+    height: 23px
+    color: #fff
+    background: $color-salmon-orange-dark2
     border: 1px solid #444
```

Comment:
This will need to be one of the new $color-simdata-carrot colors.  I removed all of the salmons as they were no longer part of the spec.

Reply:
:+1: